src/main/g8/default.properties

```diff
@@ -1,7 +1,7 @@
 name = akka-grpc-quickstart
 description = Akka gRPC is a toolkit for building streaming gRPC servers and clients on top of Akka Streams. This simple application will get you started building gRPC based systems with Java. This app uses Akka, Java, and ScalaTest.
 akka_grpc_version=0.2-RC1
-akka_version=2.5.15
+akka_version=2.5.16
```

Comment:
Use `maven(com.typesafe.akka, akka-actor_2.12)` to get the new version automatically. See http://www.foundweekends.org/giter8/template.html#Maven+properties

Reply:
Should we setup travis to build nightly in a cron job then?

Currently each edit causes a build so it's easy to track a build failure.

Reply:
for the record, I prefer the zero-maintenance, cron-backed option.

Reply:
I'll give that a go

Reply:
#11 looks good to me. :-)

Did a review there.